account/identity.go

```diff
@@ -0,0 +1,127 @@
+package account
+
+import (
+	"time"
+
+	"github.com/almighty/almighty-core/models"
+	"github.com/goadesign/goa"
+	"github.com/jinzhu/gorm"
+	uuid "github.com/satori/go.uuid"
+	"golang.org/x/net/context"
+)
+
+// Identity ddenDescribes a unique Person with the ALM
+type Identity struct {
+	models.Lifecycle
+	ID       uuid.UUID `sql:"type:uuid default uuid_generate_v4()" gorm:"primary_key"` // This is the ID PK field
+	Emails   []User    // has many Users
+	FullName string    // The fullname of the Identity
+	ImageURL string    // The image URL for this Identity
+}
+
+// TableName overrides the table name settings in Gorm to force a specific table name
+// in the database.
+func (m Identity) TableName() string {
+	return "identities"
+
+}
+
+// GormIdentityRepository is the implementation of the storage interface for
+// Identity.
+type GormIdentityRepository struct {
+	db *gorm.DB
```

Comment:
I can see other(WI, WIT, tracker) *Repository do have a `ts  *GormTransactionSupport` as a transaction support.
And methods use transaction.Create or Find etc. to read/write to DB.

I have gone through discussion over #247 And I understand that single operation does not make sense to be done in transaction.

So these methods does not use Commit or Rollback things, they just support transactions. because repo methods are not aware that if controller is going to use multiple actions together or not.
But in case of User and Identity I can see that is not the case,

Is my understanding is wrong ? Is it the responsibility of the controller(any consumer of repo methods) to make use of transactions and repo methods should not deal with any transaction object ?